lib/advisor_web/questionnaire_proposal.ex

```diff
@@ -10,6 +10,7 @@ defmodule AdvisorWeb.QuestionnaireProposal do
     field :requester, :integer
     field :questions, {:map, :boolean}
     field :advisors, {:map, :boolean}
+    field :message, :string
```

Comment:
Just wondering, what is a string limit with Ecto?

Reply:
As far as I understand it maps to `varchar(255)`